pom.xml

```diff
@@ -1465,6 +1469,20 @@
     <!-- Profiles -->
     <profiles>
         <!--
+          Name: modularizedJdk
+          Descr: various workarounds activation for modularized JDK
+        -->
+        <profile>
+            <id>modularizedJdk</id>
+            <activation>
+                <jdk>9</jdk>
+            </activation>
+            <properties>
+                <modular.jdk.args>-XaddExports:java.security.sasl/com.sun.security.sasl.digest=ALL-UNNAMED,java.security.sasl/com.sun.security.sasl=ALL-UNNAMED</modular.jdk.args>
+                <modular.jdk.props> -Dsun.util.logging.disableCallerCheck=true</modular.jdk.props>
```

Comment:
There's likely nothing the JBoss Log Manager can do. I'll have to review the JDK code again (it's recently changed quite a bit for jigsaw), but if a platform logger is used on the class with a `null` class loader it loads the JUL log manager. This happens before the system property is set IIRC.

There's a parent log manager JIRA for JDK 9 things BTW https://issues.jboss.org/browse/LOGMGR-130